k8s/scripts/vars

```diff
@@ -1,44 +1,43 @@
 # DNS name of the bigchaindb node
-NODE_FQDN="test-node.bigchaindb.com"
+NODE_FQDN="muawia-dev.ipdb.io"
 
 # Secret token used for authorization of
 # POST requests to the bigchaindb node
 SECRET_TOKEN="test-secret"
 
 # Absolute path for the SSL certificate key
-HTTPS_CERT_KEY_FILE_NAME="</path/to/https.key>"
+HTTPS_CERT_KEY_FILE_NAME="/scripts/blabla"
 
 # Absolute path for the SSL certificate chain
-HTTPS_CERT_CHAIN_FILE_NAME="</path/to/https.pem>"
+HTTPS_CERT_CHAIN_FILE_NAME="/scripts/blabla"
 
 # MongoDB Admin user credentials
 MDB_ADMIN_USER='adminUser'
 MDB_ADMIN_PASSWORD='superstrongpassword'
 
-# Tendermint instance name of the bigchaindb
-# node. This name should be unique
-TM_INSTANCE_NAME='tm-instance-0'
+# BigchainDB instance name. This name should be unique
+BDB_INSTANCE_NAME='bdb-instance-0'
 
 # Comma separated list of initial peers in the
 # network.
-TM_SEEDS='tm-instance-0,tm-instance-1,tm-instance-2'
+BDB_SEEDS='bdb-instance-0'
 
 # Comma separated list of validators in the
 # network
-TM_VALIDATORS='tm-instance-0,tm-instance-1,tm-instance-2'
+BDB_VALIDATORS='bdb-instance-0'
```

Comment:
@muawiakh How do the nodes connect if they don't share the validators?

Reply:
I updated these parameters by mistake, addressed now.